src/vm/jvm/runtime/org/raku/nqp/runtime/GlobalContext.java

```diff
@@ -4,8 +4,9 @@
 import java.io.PrintStream;
 import java.io.UnsupportedEncodingException;
 import java.lang.ref.WeakReference;
+import java.util.concurrent.CopyOnWriteArraySet;
 import java.util.HashMap;
-import java.util.HashSet;
+import java.util.Set;
```

Comment:
If I'm not mistaken, this import isn't needed after https://github.com/Kaiepi/nqp/commit/5d026b44ba